universal-application-tool-0.0.1/app/views/dev/DatabaseSeedView.java

```diff
@@ -59,7 +59,7 @@ public Content render(
                     .with(
                         form()
                             .with(makeCsrfTokenInputTag(request))
-                            .with(submitButton("Clear all programs and questions"))
+                            .with(submitButton("clear", "Clear all programs and questions"))
                             .withMethod("post")
                             .withAction(routes.DatabaseSeedController.clear().url())))
```

Comment:
Suggestions (blocking): I believe this only drops questions and programs tables. You might want to add all other tables, optionally as another button.

Reply:
Good idea - added the others.